docker-compose.yml

```diff
@@ -3,13 +3,13 @@ version: "3"
 services:
   telemetry-frontend:
     build:
-      dockerfile: frontend.Dockerfile
-      context: ./
+      dockerfile: Dockerfile
+      context: ./frontend/
     # Copy in changes to the ui, so no need to rebuild the images.
     volumes:
       - /app/node_modules
       - ./packages:/app/packages
-    command: yarn start:frontend
+    # command: yarn start:frontend
     ports:
       - 3000:3000
```

Comment:
this needs to be 3000:80 as per the nginx config running inside the container. we /could/ forgo nginx here, probably?